lib/services/presets.go

```diff
@@ -484,6 +484,70 @@ func NewPresetRequireTrustedDeviceRole() types.Role {
 	}
 }
 
+// SystemOktaAccessRoleName is the name of the system role that allows
```

Comment:
The godoc for `NewSystemOktaAccessRole` should start with `// NewSystemOktaAccessRole ...`.

Why are we discussing a role name here in a function that returns a role?

Same goes for below.